src/styles/index.css

```diff
@@ -1,9 +1,10 @@
-body {
-  margin: 0;
-  padding: 0;
-  font-family: sans-serif;
-}
+* { margin: 0; padding: 0; font-family: sans-serif; }
+#router, #root { height: 100%; }
+body { height: 0; min-height: 100%; }
 
-body, .App, #router, #root {
-  height: 100%;
-}
+.flex { display: flex; }
+.grow { flex-grow: 1; }
+.column { flex-direction: column; }
+.center { justify-content: center; align-items: center; }
+
+#app { height: 0; }
```

Comment:
@bill-feng Do you think #app `height:0` → `height: calc(100% - (20%))`?;

Reply:
so `#app` has a flex-grow on it which will make it fill the rest of the parent div. the reason to have height on it is so that the parent height property gets passed to its children.